terraform/modules/hub/variables.tf

```diff
@@ -89,6 +89,16 @@ variable "self_service_enabled" {
   default     = "false"
 }
 
+variable "cross_gov_ga_tracker_id" {
+  description = "The Google Analytics tracker ID for GOV.UK cross domain analysis"
+  default     = ""
+}
+
+variable "cross_gov_ga_domain_names" {
+  description = "List of (space delimited) domains to automatically enable links for cross-domain analytics"
```

Comment:
Also mention that this is used for SAML post forms or other cases where the user is redirected back to the original service.